Plugins/Podio/src/PodioUtil.cpp

```diff
@@ -78,7 +78,7 @@ ActsPodioEdm::Surface convertSurfaceToPodio(const ConversionHelper& helper,
     }
     result.boundValuesSize = values.size();
 
-    Eigen::Map<ActsSymMatrix<4>> trf{result.transform.data()};
```

Comment:
Why is this part of this PR, was that not caught bey the CI?

Reply:
I just now updated this PR from a few weeks ago and had to update a few files.